modules/objdetect/src/qrcode.cpp

```diff
@@ -0,0 +1,760 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2017, Intel Corporation, all rights reserved.
+// Third party copyrights are property of their respective owners.
+
+#include "precomp.hpp"
+#include "qrcode.hpp"
+// #include <opencv2/calib3d/calib3d.hpp>
+
+#include <limits>
+#include <cmath>
+#include <iostream>
+
+using namespace std;
```

Comment:
do not use `using namespace ...` inside OpenCV .cpp files. Put everything into `namespace cv { }`. Add `using std::vector;` etc. for each of the major type of function that you use many times; or just add `std::` for some functions and classes that you use just few times